.sail/Dockerfile

```diff
@@ -13,4 +15,6 @@ RUN wget -O /tmp/hugo.deb https://github.com/gohugoio/hugo/releases/download/v0.
   sudo dpkg -i /tmp/hugo.deb && \
   rm -f /tmp/hugo.deb
 
-RUN installext peterjausovec.vscode-docker
+# Fails to install Extension 'ms-azuretools.vscode-docker' not found.
+#    error vscode undefined
+# RUN installext ms-azuretools.vscode-docker
```

Comment:
Just remove this, it's not a super-important extension anyways since we manage docker stuff from cli